src/components/portrait/portrait.test.js

```diff
@@ -259,4 +261,173 @@ context("Tests for Portrait component", () => {
       });
     });
   });
+
+  describe("Accessibility tests for Portrait component", () => {
+    it("should check gravatar for accessibility tests", () => {
+      CypressMountWithProviders(
+        <PortraitDefaultComponent gravatar="chris.barber@sage.com" />
+      );
+      cy.checkAccessibility();
+    });
+
+    it.each(imageURL)(
+      "should check custom image url for accessibility tests",
+      (srcImage) => {
+        CypressMountWithProviders(<PortraitDefaultComponent src={srcImage} />);
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([
+      PORTRAIT_SIZES[0],
+      PORTRAIT_SIZES[1],
+      PORTRAIT_SIZES[2],
+      PORTRAIT_SIZES[3],
+      PORTRAIT_SIZES[4],
+      PORTRAIT_SIZES[5],
+      PORTRAIT_SIZES[6],
+    ])("should check %s size for accessibility tests", (size) => {
+      CypressMountWithProviders(<PortraitDefaultComponent size={size} />);
+      cy.checkAccessibility();
+    });
+
+    it("should check alt text for accessibility tests", () => {
+      CypressMountWithProviders(
+        <PortraitDefaultComponent alt="cypress-test" />
+      );
+      cy.checkAccessibility();
+    });
+
+    it.each(["square", "circle"])(
+      "should check %s shape for accessibility tests",
+      (shape) => {
+        CypressMountWithProviders(<PortraitDefaultComponent shape={shape} />);
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["error", "warning", "info"])(
+      "should check %s icon for accessibility tests",
+      (iconType) => {
+        CypressMountWithProviders(
+          <PortraitDefaultComponent iconType={iconType} />
+        );
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["KW", "JC"])(
```

Comment:
Portrait can also accept three initials. 
@DlgSHi - Do you think we need to have a test case for three initials too?

Reply:
@DipperTheDan I'll add a test case for three initials.